CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
+## 5.1.0
+* `alphabetical` sorting now puts shorthands before their longhand forms even if that isn't alphabetical.
```

Comment:
```suggestion
* Changed: `properties-order: "alphabetical"` now puts shorthands before their longhand forms even if that isn't alphabetical to avoid broken CSS. E. g. `border-color` will be before `border-bottom-color`.
```